lib/types/webp.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs      = require('fs');
+var Promise = require('bluebird');
+var pread   = Promise.promisify(fs.read);
+var utils   = require('../utils');
+
+function isWEBP (buffer) {
+  return 'RIFF' === utils.ascii(buffer, 0, 4) && 'WEBP' === utils.ascii(buffer, 8, 12);
```

Comment:
ahh i see you were following my lead with the yoda-style conditionals, but that should probably change back. I think they are generally funky and harder to read. I'll put up a PR to convert my yoda statements.